pipeline/azure-pipelines.yml

```diff
@@ -104,11 +99,32 @@ jobs:
       retryCount: 5
 
   - task: DownloadSecureFile@1
-    name: pfx
-    displayName: Download pfx file
+    name: SM_CLIENT_CERT
+    displayName: Download p12 file
     inputs:
-      secureFile: SonarSource-2021-2023.pfx
+      secureFile: digicert_authentication_certificate.p12
       retryCount: 5
+  
+  - task: SSMClientToolsSetup@1
+    displayName: Client Tools Setup
+    condition: eq(variables['SHOULD_SIGN'], 'true')
```

Comment:
I'm not sure I understand who sets `SHOULD_SIGN` to true and when this happens. Could give me a hint? :)

Reply:
https://github.com/SonarSource/sonarlint-visualstudio/blob/master/pipeline/scripts/should-sign-vsix.ps1#L43

Reply:
Ok, now I got it. Thanks!